app.js

```diff
@@ -237,5 +240,20 @@ submit.addEventListener('submit', function (e) {
     resultMsg.textContent = 'Email must be in lower case';
     resultMsg.style.color = 'red';
     return false;
+  } else {
+    const formData = {
+      name: nameInput.value,
+      email: emailInput.value,
+      message: msgInput.value,
+    };
```

Comment:
Kindly make sure that when the user changes the content of any input field, the data is saved to the local storage. This is a project requirement. :+1: :smiley: :smiley:  It means that the contact form information is persistent to page reloads.
and after filling the form, and not submitting it, the entered values should be saved in the browser. 

![image](https://user-images.githubusercontent.com/98363075/195612054-d886ae45-946e-4d98-97d6-80f39f49f33b.png)